### PR TITLE
Azure update network arm

### PR DESCRIPTION
**Pull request by yogevh**
Verify Acceptance tests before approving PR

### CODE REVIEW

File: azure/pas-dr-deploy.json [hunk 1]
```diff
@@ -108,16 +108,42 @@
...
         "Vault DR AvailablitySet Name": {
             "type": "String",
             "metadata": {
                 "description": "Enter the Availability Set for the Vault DR VM , If doesn’t exist it will be created"
             }
-        }
+        },
+        "DR Vault Vnet Name":{
+            "type":"String",
+            "defaultValue": "Pas-Vnet",
```
Comment (yogevh) [inline azure/pas-dr-deploy.json]: change to PAS-Vnet (also change the name in network deployment)

File: azure/pas-dr-deploy.json [hunk 2]
```diff
@@ -108,16 +108,42 @@
...
+            "metadata":{
+               "description":"Enter the Vnet for the Vault-DR VM"
+            }
+         },
+         "DR Vault Subnet Name":{
+            "type":"String",
+            "defaultValue": "Vault-Subnet",
+            "metadata":{
+               "description":"Enter the Vnet for the Vault-DR VM"
```
Comment (yogevh) [inline azure/pas-dr-deploy.json]: change vnet to Subnet

Comment (yogevh) [inline azure/pas-dr-deploy.json]: VNet

File: azure/pas-full-network.json [hunk 3]
```diff
@@ -2,53 +2,53 @@
...
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "Vault VNET CIDR": {
+    "PAS VNET CIDR": {
       "type": "string",
-      "defaultValue": "10.0.1.0/24",
+      "defaultValue": "10.0.0.0/16",
       "metadata": {
-        "description": "IPv4 address range for the Vault VNET"
+        "description": "IPv4 address range for the Solution VNET"
```
Comment (yogevh) [inline azure/pas-full-network.json]: PAS VNet

File: azure/pas-full-network.json [hunk 4]
```diff
@@ -2,53 +2,53 @@
...
       }
     },
-    "Vault Main Subnet CIDR": {
+    "Vault Subnet CIDR": {
       "type": "string",
-      "defaultValue": "10.0.1.0/28",
+      "defaultValue": "10.0.1.0/24",
       "metadata": {
         "description": "IPv4 address range for the main Vault subnet"
```
Comment (yogevh) [inline azure/pas-full-network.json]: remove main

File: azure/pas-full-network.json [hunk 5]
```diff
@@ -2,53 +2,53 @@
...
       }
     },
-    "Components Subnet CIDR": {
+    "PTA Subnet CIDR": {
       "type": "string",
-      "defaultValue": "10.10.2.0/26",
+      "defaultValue": "10.0.6.0/24",
       "metadata": {
-        "description": "IPv4 address range for the second private Components subnet"
+        "description": "IPv4 address range for the PSM subnet"
```
Comment (yogevh) [inline azure/pas-full-network.json]: change to PTA

File: azure/pas-full-network.json [hunk 6]
```diff
@@ -2,53 +2,53 @@
...
       }
     },
-    "Web Subnet CIDR": {
+    "PSMP Subnet CIDR": {
       "type": "string",
-      "defaultValue": "10.10.1.0/26",
+      "defaultValue": "10.0.5.0/24",
       "metadata": {
-        "description": "IPv4 address range for the first private Components subnet"
+        "description": "IPv4 address range for the PSM subnet"
```
Comment (yogevh) [inline azure/pas-full-network.json]: change to PSMP

File: azure/pas-full-network.json [hunk 7]
```diff
@@ -343,41 +261,60 @@
             }
           },
           {
-            "name": "AllowRdpUserAccessIn",
+            "name": "AllowSshOut",
```
Comment (yogevh) [inline azure/pas-full-network.json]: remove rule

File: azure/pas-full-network.json [hunk 8]
```diff
@@ -657,126 +565,571 @@
...
-          "id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('networkSecurityGroups_NSG_Web_name'))]"
-        },
-        "serviceEndpoints": []
-      },
-      "dependsOn": [
-        "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworks_Components_Vnet_name'))]",
-        "[resourceId('Microsoft.Network/networkSecurityGroups', variables('networkSecurityGroups_NSG_Web_name'))]"
-      ]
-    },
+        "securityRules": [
```
Comment (yogevh) [inline azure/pas-full-network.json]: remove

File: azure/pas-full-network.json [hunk 9]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "AllowSshUserAccessIn",
+            "properties": {
+              "protocol": "TCP",
+              "sourcePortRange": "*",
+              "destinationPortRange": "22",
+              "sourceAddressPrefix": "[parameters('User Access CIDR')]",
```
Comment (yogevh) [inline azure/pas-full-network.json]: change to administrative access

File: azure/pas-full-network.json [hunk 10]
```diff
@@ -657,126 +565,571 @@
...
+              "priority": 120,
+              "direction": "Inbound",
+              "sourcePortRanges": [],
+              "destinationPortRanges": [],
+              "sourceAddressPrefixes": [],
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "AllowHttpsUserAccessIn",
```
Comment (yogevh) [inline azure/pas-full-network.json]: remove

File: azure/pas-full-network.json [hunk 11]
```diff
@@ -657,126 +565,571 @@
...
+              "priority": 130,
+              "direction": "Inbound",
+              "sourcePortRanges": [],
+              "destinationPortRanges": [],
+              "sourceAddressPrefixes": [],
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "AllowRdpUserAccessIn",
```
Comment (yogevh) [inline azure/pas-full-network.json]: remove

File: azure/pas-full-network.json [hunk 12]
```diff
@@ -657,126 +565,571 @@
...
+              "priority": 140,
+              "direction": "Inbound",
+              "sourcePortRanges": [],
+              "destinationPortRanges": [],
+              "sourceAddressPrefixes": [],
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "AllowRDPAdministrativeAccessIn",
```
Comment (yogevh) [inline azure/pas-full-network.json]: remove

File: azure/pas-full-network.json [hunk 13]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "AllowSshOut",
+            "properties": {
+              "protocol": "TCP",
+              "sourcePortRange": "*",
+              "destinationPortRange": "22",
+              "sourceAddressPrefix": "[parameters('PVWA Subnet CIDR')]",
```
Comment (yogevh) [inline azure/pas-full-network.json]: remove source

File: azure/pas-full-network.json [hunk 14]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefix": "*",
```
Comment (yogevh) [inline azure/pas-full-network.json]: destination should be open to all on port 22

File: azure/pas-full-network.json [hunk 15]
```diff
@@ -657,126 +565,571 @@
...
+              "priority": 110,
+              "direction": "Outbound",
+              "sourcePortRanges": [],
+              "destinationPortRanges": [],
+              "sourceAddressPrefixes": [],
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "AllowRdpOut",
```
Comment (yogevh) [inline azure/pas-full-network.json]: remvoe

File: azure/pas-full-network.json [hunk 16]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "Allow1858VaultOut",
+            "properties": {
+              "protocol": "TCP",
+              "sourcePortRange": "*",
+              "destinationPortRange": "1858",
+              "sourceAddressPrefix": "[parameters('PVWA Subnet CIDR')]",
```
Comment (yogevh) [inline azure/pas-full-network.json]: change source to PSMP cidr

File: azure/pas-full-network.json [hunk 17]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefix": "",
```
Comment (yogevh) [inline azure/pas-full-network.json]: change destinatoin to Vault cidr

File: azure/pas-full-network.json [hunk 18]
```diff
@@ -657,126 +565,571 @@
...
       "dependsOn": [
-        "[resourceId('Microsoft.Network/networkSecurityGroups', variables('networkSecurityGroups_NSG_Vault_name'))]",
-        "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworks_Components_Vnet_name'))]"
+        "[resourceId('Microsoft.Network/networkSecurityGroups', variables('Vault_NSG_name'))]"
       ]
     },
     {
-      "type": "Microsoft.Network/virtualNetworks",
-      "name": "[variables('virtualNetworks_Vault_DR_Vnet_name')]",
+      "comments": "PSMP Components Security Group",
```
Comment (yogevh) [inline azure/pas-full-network.json]: Add new rule - outbound on port 443 to all

File: azure/pas-full-network.json [hunk 19]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "PTA_TCP_out_3",
+            "properties": {
+              "protocol": "TCP",
+              "sourcePortRange": "80",
+              "destinationPortRange": "80",
+              "sourceAddressPrefix": "[parameters('PVWA Subnet CIDR')]",
```
Comment (yogevh) [inline azure/pas-full-network.json]: replace source with destination

File: azure/pas-full-network.json [hunk 20]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "PTA_TCP_out_4",
+            "properties": {
+              "protocol": "TCP",
+              "sourcePortRange": "1858",
+              "destinationPortRange": "1858",
+              "sourceAddressPrefix": "[parameters('Vault Subnet CIDR')]",
```
Comment (yogevh) [inline azure/pas-full-network.json]: replace source with desitnation

File: azure/pas-full-network.json [hunk 21]
```diff
@@ -657,126 +565,571 @@
...
+            }
+          },
+          {
+            "name": "PTA_UDP_Out_1",
+            "properties": {
+              "protocol": "UDP",
+              "sourcePortRange": "",
+              "destinationPortRange": "",
+              "sourceAddressPrefix": "0.0.0.0/0",
+              "destinationAddressPrefix": "[parameters('PTA Subnet CIDR')]",
```
Comment (yogevh) [inline azure/pas-full-network.json]: replace

File: azure/pas-full-network.json [hunk 22]
```diff
@@ -657,126 +565,571 @@
...
+              "destinationAddressPrefixes": []
+            }
+          },
+          {
+            "name": "PTA_UDP_Out_2",
+            "properties": {
+              "protocol": "UDP",
+              "sourcePortRange": "1858",
+              "destinationPortRange": "1858",
+              "sourceAddressPrefix": "[parameters('Vault Subnet CIDR')]",
```
Comment (yogevh) [inline azure/pas-full-network.json]: replace

File: azure/pas-vault-deploy.json [hunk 23]
```diff
@@ -97,18 +98,44 @@
...
+         "defaultValue": "Pas-Vnet",
+         "metadata":{
+            "description":"Enter the Vnet for the Vault VM"
+         }
+      },
+      "DR Vault Vnet Name":{
+         "type":"String",
+         "defaultValue": "Pas-Vnet",
+         "metadata":{
+            "description":"Enter the Vnet for the Vault-DR VM"
```
Comment (yogevh) [inline azure/pas-vault-deploy.json]: change cnet to subnet

File: azure/pas-vault-deploy.json [hunk 24]
```diff
@@ -97,18 +98,44 @@
...
          "type":"String",
          "metadata":{
             "description":"Enter the Availability Set for the Vault VM , If doesn’t exist it will be created"
          }
+      },
+      "Primary Vault Vnet Name":{
+         "type":"String",
+         "defaultValue": "Pas-Vnet",
+         "metadata":{
+            "description":"Enter the Vnet for the Vault VM"
```
Comment (yogevh) [inline azure/pas-vault-deploy.json]: change Vnet to VNet

File: azure/pas-vault-deploy.json [hunk 25]
```diff
@@ -97,18 +98,44 @@
...
+         }
+      },
+      "DR Vault Vnet Name":{
+         "type":"String",
+         "defaultValue": "Pas-Vnet",
```
Comment (yogevh) [inline azure/pas-vault-deploy.json]: PAS VNet

File: azure/pas-vault-deploy.json [hunk 26]
```diff
@@ -97,18 +98,44 @@
...
+      },
       "Vault AvailablitySet Name":{
          "type":"String",
          "metadata":{
             "description":"Enter the Availability Set for the Vault VM , If doesn’t exist it will be created"
          }
+      },
+      "Primary Vault Vnet Name":{
+         "type":"String",
+         "defaultValue": "Pas-Vnet",
```
Comment (yogevh) [inline azure/pas-vault-deploy.json]: PAS VNet

File: azure/pas-vault-deploy.json (hunk 23 above)
Comment (yogevh) [inline azure/pas-vault-deploy.json]: VNet

File: aws/PAS-Component-Single-Deployment.json [hunk 27]
```diff
@@ -2390,5 +2390,9 @@
             "Ref":"LogGroup"
          }
       }
+   },
+   "GetVaultMachineIp":{
+    "Description": "PrivateIP",
+    "Value" : { "Fn::GetAtt" : [ "PVWAMachine", "PrivateIp" ]}
    }
 }
```
Comment (avishayil) [inline aws/PAS-Component-Single-Deployment.json]: How is this related to azure? is it for the cross cloud feature?

File: azure/pas-full-network.json [hunk 28]
```diff
@@ -67,28 +67,25 @@
...
-    "networkSecurityGroups_NSG_Components_name": "NSG-Components",
-    "subnets_Web_Subnet_name": "Web-Subnet",
-    "networkSecurityGroups_NSG_Web_name": "NSG-Web",
-    "virtualNetworkPeerings_Vault_Components_Peering_name": "Vault-Components-Peering",
-    "virtualNetworkPeerings_Components_Vault_Peering_name": "Components-Vault-Peering",
-    "virtualNetworkPeerings_Vault_VaultDR_Peering_name": "Vault-VaultDR-Peering",
-    "virtualNetworkPeerings_VaultDR_Vault_Peering_name": "VaultDR-Vault-Peering",
-    "virtualNetworkPeerings_VaultDR_Components_Peering_name": "VaultDR-Components-Peering",
-    "virtualNetworkPeerings_Components_VaultDR_Peering_name": "Components-VaultDR-Peering"
+    "virtualNetworks_Solution_VNet_name": "PAS-VNet",
```
Comment (avishayil) [inline azure/pas-full-network.json]: Why subnets are beginning with the component name, and NSG beginning with NSG?
Please be consistent?

File: azure/pas-full-network.json [hunk 29]
```diff
@@ -107,82 +104,40 @@
...
               "destinationAddressPrefixes": [
-                "[parameters('Vault Main Subnet CIDR')]",
-                "[parameters('Vault DR Subnet CIDR')]"
+                "[parameters('Vault Subnet CIDR')]"
               ]
             }
           },
           {
-            "name": "Allow1858VaultDRIn",
+            "name": "AllowRabbitPVWA",
```
Comment (avishayil) [inline azure/pas-full-network.json]: Rabbit?

File: azure/pas-full-network.json [hunk 30]
```diff
@@ -557,33 +562,79 @@
...
+              "description": "Deny all outbound traffic",
+              "protocol": "*",
+              "sourcePortRange": "*",
+              "destinationPortRange": "*",
+              "sourceAddressPrefix": "*",
+              "destinationAddressPrefix": "*",
+              "access": "Deny",
+              "priority": 65500,
+              "direction": "Outbound",
+              "sourcePortRanges": [],
```
Comment (avishayil) [inline azure/pas-full-network.json]: Empty arrays?

File: azure/pas-full-network.json [hunk 31]
```diff
@@ -594,7 +645,45 @@
             }
           },
           {
-            "name": "DenyAllOut",
+            "name": "Allow1858VaultOut",
+            "properties": {
+              "protocol": "TCP",
+              "sourcePortRange": "*",
```
Comment (avishayil) [inline azure/pas-full-network.json]: Why source port range is *? Do we want to allow using another source port for communication with the vault?

File: azure/pas-full-network.json [hunk 32]
```diff
@@ -617,6 +944,7 @@
           {
             "name": "DenyAllInBound",
             "properties": {
+              "provisioningState": "Succeeded",
```
Comment (avishayil) [inline azure/pas-full-network.json]: Why is this needed?

File: azure/pas-single-component-deploy.json [hunk 33]
```diff
@@ -89,19 +89,23 @@
...
         },
-        "Component AvailablitySet Name": {
-            "type": "String",
-            "metadata": {
-                "description": "Enter the Availability Set for the Component VM , If doesn’t exist it will be created"
+        "Availability Zone":{
+            "type":"string",
+            "allowedValues": [
+                "1",
+                "2",
```
Comment (avishayil) [inline azure/pas-single-component-deploy.json]: 4 is not allowed?

File: azure/pas-full-network.json (hunk 29 above)
Comment (NoamALK) [reply azure/pas-full-network.json]: Yep. RabbitMQ i think

File: azure/pas-single-component-deploy.json (hunk 33 above)
Comment (NoamALK) [reply azure/pas-single-component-deploy.json]: it is allowed but there are regions where 4 isnt available. we decided 3 is more than enough

File: azure/pas-full-network.json (hunk 28 above)
Comment (NoamALK) [reply azure/pas-full-network.json]: Done.

File: azure/pas-full-network.json (hunk 4 above)
Comment (NoamALK) [reply azure/pas-full-network.json]: Done

File: aws/PAS-Component-Single-Deployment.json (hunk 27 above)
Comment (NoamALK) [reply aws/PAS-Component-Single-Deployment.json]: Removed

File: azure/pas-full-network.json (hunk 32 above)
Comment (NoamALK) [reply azure/pas-full-network.json]: done

File: azure/pas-full-network.json (hunk 30 above)
Comment (NoamALK) [reply azure/pas-full-network.json]: Done

**Comment by avishayil** [comment]
Please fix the conflicts @NoamALK
